R/calcACOwnershipRatesFilled.R

```diff
@@ -1,9 +1,16 @@
-#' Calculate Filled Air Conditioning Ownership Rates with Regional Parameters
+#' Calculate Filled Air Conditioning Ownership Rates with Regional Calibration
```

Comment:
I was wondering what _Filled Air_ could mean and was irritated :)
```suggestion
#' Calculate Extrapolated Air Conditioning Ownership Rates with Regional Calibration
```